src/components/language-context.js

```diff
@@ -15,9 +15,9 @@ import {
   UI_LANGUAGE_COOKIE_EXPIRES,
   getLocale
 } from '../resources/ui-language.js';
-
-import 'moment';
-import 'moment/locale/de.js';
+import { format, parseISO } from 'date-fns';
+import deDeLocale from 'date-fns/locale/de/index.js';
+import enUsLocale from 'date-fns/locale/en-US/index.js';
 
 const enUS = enUSNs.default || enUSNs;
 const deDE = deDENs.default || deDENs;
```

Comment:
Gets a little confusing with 2 sets of locale variables here.
An idea would be to maybe make 2 clearly named groups, like
```
const antLocales = {
  enUS: enUSNs.default || enUSNs;
  deDE: deDENs.default || deDENs;
};

const dateLocales = {
 enUS: enUsLocale,
 deDE: deDeLocale,
}
```
What do you think?

Reply:
Done